### PR TITLE
feat: add the field `is_from_offline` to the class `Message`

### PR DESCRIPTION
**Pull request by mahdyar**

### CODE REVIEW

File: tests/test_message.py [hunk 1]
```diff
@@ -263,6 +263,7 @@ def message(bot):
         {"reply_to_story": Story(Chat(1, Chat.PRIVATE), 0)},
         {"boost_added": ChatBoostAdded(100)},
         {"sender_boost_count": 1},
+        {"is_from_offline": True}
```
Comment (harshil21) [inline tests/test_message.py:266]:
```suggestion
        {"is_from_offline": True},
```

File: telegram/_message.py [hunk 2]
```diff
@@ -888,6 +894,7 @@ class Message(MaybeInaccessibleMessage):
         "group_chat_created",
         "has_media_spoiler",
         "has_protected_content",
+        "is_from_offline",
```
Comment (harshil21) [inline telegram/_message.py:897]: please run `ruff check telegram/ --fix` (see contrib guide on running pre-commit)

**Review by harshil21 — CHANGES_REQUESTED** [review]
thanks for the quick pr!

**Comment by mahdyar** [comment]
> thanks for the quick pr!

Thx for the quick review! :)

**Comment by harshil21** [comment]
One more thing - you also need to add an entry to `constants.MessageType` for the same field. And then make sure test_constants.py passes

**Comment by harshil21** [comment]
> One more thing - you also need to add an entry to `constants.MessageType` for the same field. And then make sure test_constants.py passes

Update: Apparently you shouldn't add it to constants. Instead add an ignore entry to that field in the [test](https://github.com/python-telegram-bot/python-telegram-bot/blob/3ff7a6896c46c4efcd43c03bfd4525f2bc7a037b/tests/test_constants.py#L152)

**Comment by mahdyar** [comment]
> > One more thing - you also need to add an entry to `constants.MessageType` for the same field. And then make sure test_constants.py passes
> 
> Update: Apparently you shouldn't add it to constants. Instead add an ignore entry to that field in the [test](https://github.com/python-telegram-bot/python-telegram-bot/blob/3ff7a6896c46c4efcd43c03bfd4525f2bc7a037b/tests/test_constants.py#L152)

Done.

**Comment by harshil21** [comment]
Test failures are unrelated. Thanks for contributing!

**Pull request merged**